_includes/tutorials/versioned-ktables/kstreams/markup/dev/short-answer.adoc

```diff
@@ -0,0 +1,21 @@
+You can run your application with link:https://www.confluent.io/confluent-cloud/tryfree/[Confluent Cloud].
+
+
+In the Kafka Streams application use a `VersionedKeyValueStore` when creating your `KTable` the retention time shown is just
```

Comment:
three sentences unless you reword, or em dash the final sentence, i.e.:

>In the Kafka Streams application, use a `VersionedKeyValueStore` when creating your `KTable`. The retention time shown is just a placeholder — you are free to use any time that suits your event streaming requirements.

Reply:
ack